yazi-scheduler/Cargo.toml

```diff
@@ -22,6 +22,7 @@ futures                = "0.3.30"
 parking_lot            = "0.12.3"
 scopeguard             = "1.2.0"
 tokio                  = { version = "1.38.0", features = [ "full" ] }
+walkdir                = "2.5.0"
```

Comment:
Please remove the `walkdir` crate, I don't want to introduce new dependencies, and `walkdir` uses sync I/O instead of async I/O.

Reply:
Done!